### ISSUE TITLE
Change "Please answer Yes or No"

### ISSUE THREAD

**Issue by luketlancaster** [ldmcdaniel/dotfiles#2]
On lines like these:

https://github.com/ldmcdaniel/dotfiles/blob/master/.scripts/install_zsh.sh#L12

Change it to `Please answer Y/y or N/n' (or something). Unless I'm off, "Yes" != "y", right?

**Comment by ldmcdaniel** [comment]
True.  That would be clearer.  I don't know a lot of shell script, but I thought `[Yy]*` used the wildcard and would accept `y` or `yes` or `yeah`.  That was my thinking, although unconfirmed.  I will look into it.  Thanks for the suggestions!
